.github/workflows/_meta.yaml

```
@@ -92,7 +92,7 @@ jobs:
       - name: Prepare Release Assets
         run: |-
           pushd artifact
-          for file in *.deb *.zip; do
+          find * -type f \( -name "*.deb" -o -name "*.zip" \) | while read file; do
```

While I would hate whoever made uppercase bits in these file extensions, it may be worth using `-iname` for case insensitive filename matching.

There is also a difference between `find *` and `find .`, dotfiles are not matched by `*`.
Not that we'd ever want dotfiles as actual artifacts tho, just pointing it out.

```suggestion
          find * -type f \( -iname "*.deb" -or -iname "*.zip" \) | while read file; do
```
I prefer the long version of or, as it's more clear on what it does without being familiar with find (but it isn't POSIX, which shouldn't matter as we're running this under Ubuntu).

---

I agree on using `iname` but the use of `*` is intentional to strip the leading `./` from the result of find.